scalafix-tests/unit/src/test/scala/scalafix/tests/config/ArgsSuite.scala

```diff
@@ -2,9 +2,9 @@ package scalafix.tests.config
 
 import metaconfig.Conf
 import metaconfig.internal.ConfGet
-import scalafix.internal.v1.Args
 import metaconfig.typesafeconfig.typesafeConfigMetaconfigParser
 import scalafix.internal.config.ScalafixConfig
+import scalafix.internal.v1.Args
```

Comment:
Probably a side-effect of https://github.com/scalacenter/scalafix/pull/1279#issuecomment-725698604. So it looks like scalafix is still not applied/enforced anywhere, I'll follow up in a separate PR /cc @mlachkar